src/libraries/System.IO.FileSystem/tests/File/EncryptDecrypt.cs

```diff
@@ -43,9 +43,11 @@ public static void EncryptDecrypt_Read()
                 {
                     File.Encrypt(tmpFileName);
                 }
-                catch (IOException e) when (e.HResult == unchecked((int)0x80070490))
+                catch (IOException e) when (e.HResult == unchecked((int)0x80070490) ||
+                                           (e.HResult == unchecked((int)0x80071776)))
```

Comment:
I'm guessing this is the workaround you called out in the PR description? Do we want to merge this in? Or did you have it here just to make CI pass?

Reply:
Yes, I would like to merge it in. It didn't seem worth its own PR.